nemo/llama2-7b/vertex-payload.json

```diff
@@ -29,6 +29,18 @@
               {
                 "name": "NNODES",
                 "value": "4"
+              },
+              {
+                "name": "CONFIG_PATH",
+                "value": "/workspace/dist-training-vertex/nemo/llama2-7b/"
+              },
+              {
+                "name": "CONFIG_NAME",
+                "value": "llama2-7b.yaml"
```

Comment:
can this file be moved up one level as well and model name is used to call the different configs. Additional_args in each respective directory is what gets added to the job dynamically.